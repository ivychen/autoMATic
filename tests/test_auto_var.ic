int main() {
    auto x = 5;
    y := "a string";

    print(5);
    print(y);

    y = "another string";
    print(y);
}
