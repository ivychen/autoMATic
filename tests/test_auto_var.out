5
a string
another string
